mesmerize_viz/_cnmf.py

```diff
@@ -739,25 +739,26 @@ def _set_data(self, data_arrays: Dict[str, np.ndarray]):
             for g in self._image_widget.managed_graphics:
                 g.registered_callbacks.clear()
 
-            for subplot in self._image_widget.gridplot:
+            for subplot in self._image_widget.figure:
                 if "contours" in subplot:
                     # delete the contour graphics
                     subplot.delete_graphic(subplot["contours"])
 
         # absolute garbage monkey patch which I will fix once we make ImageWidget emit its own events
```

Comment:
my garbage monkeypatch isn't required anymore since you can now do [`ImageWidget.add_event_handler()`](https://www.fastplotlib.org/ver/dev/api/widgets/ImageWidget_api/fastplotlib.ImageWidget.add_event_handler.html#fastplotlib.ImageWidget.add_event_handler) :smile:

Reply:
you can probably add the image widget to the store model :thinking: